doc/README.rst

```diff
@@ -2,8 +2,8 @@
 Documentation Source
 ####################
 
-This folder contains source files for the documentation. The sources do not render well in GitHub or an RST viewer.
-In addition, some information is not visible at all and some links will not be functional.
+This folder contains source files for the documentation and is intended for XMOS users. Pre-built documentation is published on `the XMOS website<https://www.xmos.com>`_.
```

Comment:
Today in the Software Coordination Meeting chat, it was pointed out that XMOS users are external ones too, Angel suggested we use employees instead of users. I don't see a big issue with going with users, we have used that text in several repos already.